internal/ansible/proxy/kubeconfig/kubeconfig.go

```diff
@@ -68,18 +68,25 @@ type NamespacedOwnerReference struct {
 	Namespace string
 }
 
+func EncodeOwnerRef(ownerRef metav1.OwnerReference, namespace string) (string, error) {
```

Comment:
Add a comment about what this does and why you can't just marhsal `NamespacedOwnerReference`.

Reply:
yeah that seems fair